.travis.yml

```diff
@@ -13,10 +13,16 @@ cache:
     - $HOME/.cargo
     - $TRAVIS_BUILD_DIR/target
 
-before_script:
-  # TODO: Installing mdbook from crates.io takes quite a long time. This could
-  # be reduced by downloading the release from GitHub directly.
-  - mdbook -V || cargo install mdbook --vers '=0.3.1'
+before_install:
+  - sudo apt-get update
+  - sudo apt-get -y install jq curl
+  # Retrieve latest mdbook release.
+  - RELEASES="$(curl https://api.github.com/repos/rust-lang/mdbook/releases/latest)"
```

Comment:
Note that this is likely to hit a rate limit: unauthenticated calls are limited by GitHub to 60 requests per hour per IP, and if I remember correctly Travis CI uses NAT to share the same IP between multiple builders.

Reply:
Well since this in before_install shouldn't it only happen once per PR? Or do you mean that we'd get spurious failures?

Reply:
We could get spurious failures, if builds from other people with the same IP address already exausted the (small) hourly quota.

Reply:
I don't know if it solves this problem, but I decided to make the PR fully switch over to GitHub Actions.

Reply:
The problem sort of exists with GitHub Actions as well, but the upside of using GHA is that you have a token available in `${{ secrets.GITHUB_TOKEN }}`.

Reply:
That's true. I'll make that change. Would you know why this PR hasn't caused a GitHub Actions to build?

Reply:
Hmm, it should have...